Dangerfile

```diff
@@ -1,5 +1,11 @@
 # frozen_string_literal: true
 
+# Skip Danger run if the opened pull request has "wip" label, [WIP] in the title, or status is Draft
+if github.pr_labels.include?('wip') || github.pr_title.include?('[WIP]') || github.pr_json['draft']
```

Comment:
`github.pr_labels.include?('wip')`

The label could be: `type : wip` -> it does not match with this check. I think using Thanh's suggestion is safety.

```ruby
github.pr_labels.any? { |label| label.include? "wip" }
```

Reply:
Also, we can call the `github.pr_draft?` directly 😜 [(ref)](https://github.com/danger/danger/blob/master/lib/danger/danger_core/plugins/dangerfile_github_plugin.rb#L151C9-L153)

Reply:
Yeah. That makes more sense. Fixed in 9a74913d2b9b861f7abb0fcb1d0635036a3937a1